README.md

```diff
@@ -1,14 +1,12 @@
-[![Build Status](https://api.travis-ci.com/t-900-a/moneroodoo.svg?branch=main)](https://travis-ci.com/t-900-a/moneroodoo)
-[![codecov](https://codecov.io/gh/t-900-a/moneroodoo/branch/main/graph/badge.svg?token=10S5GGNRHH)](https://codecov.io/gh/t-900-a/moneroodoo)
-# Monero addons for Odoo
-
-Allows you to accept Monero as Payment within your Odoo Ecommerce shop
-
-![Monero](https://raw.githubusercontent.com/t-900-a/moneroodoo/dev/monero-rpc-odoo/static/src/img/logo.png)
-
-
 Available addons
 ----------------
 |  addon | version  | summary  |
 |---|---|---|
-|  [monero-rpc-odoo](monero-rpc-odoo/) |  14.0.1.0.2 |  Accept Monero Payment via a Wallet RPC |
+|  [monero-rpc-odoo](monero-rpc-odoo/) |  18.0.0.0.1 |  Accept Monero Payment via a Wallet RPC |
```

Comment:
This upgrade now works for v15 to v18 as the architecture has remained unchanged since v15